search_service/elastic/search.py

```diff
@@ -119,10 +119,10 @@ def __get_every_issue_id(protocol, host, port):
     """
     issue_ids = []
     uids = send_request_to_graphql(query_all_uid(), protocol, host, port)
-
-    for id in uids.get("issues"):
-        if id not in issue_ids:
-            issue_ids.append(id.get("uid"))
+    if uids:
+        for id in uids.get("issues"):
+            if id not in issue_ids:
```

Comment:
I don't get this line... You are checking if `id`, which seems to be some kind of dictionary, is not in your issue_ids. But then you only add pure uids, which seem to be integers. How is it possible, that your id-dict can be in issue_ids?